submissions/serializers.py

```diff
@@ -221,3 +221,18 @@ class Meta:
             'submission_uuid',
             'annotations',
         )
+
+
+class SubmissionListSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Submission
+        fields = '__all__'
```

Comment:
Please explicitly list the fields here instead of grabbing everything from the model. It's more verbose, but new fields in the `Submission` shouldn't automatically ripple through to the views. We need to be really careful about anything that might accidentally break backwards compatibility.

Reply:
Done